FEXCore/Source/Interface/Core/Interpreter/Fallbacks/F80Fallbacks.h

```diff
@@ -36,6 +36,14 @@ FEXCORE_PRESERVE_ALL_ATTR static softfloat_state SoftFloatStateFromFCW(uint16_t
   return State;
 }
 
+// Helper function to propagate SoftFloat exceptions to X87 flags
+FEXCORE_PRESERVE_ALL_ATTR static void HandleX87Exception(const softfloat_state& State, FEXCore::Core::CpuStateFrame* Frame) {
```

Comment:
Thoughts on making an RAII object with this and SoftFloatStateFromFCW? Would allow e.g.:

ScopedSoftFloatState State{FCW, Frame}
return X80SoftFloat(src).ToI32(&State.State);